docs/samples/git-setup/clustertemplates/version_4.Y.Z+1/sno-ran-du/sno-ran-du-v4-Y-Z-1.yaml

```diff
@@ -0,0 +1,286 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
```

Comment:
Is this for seed cluster deployment? I wonder if it might be better not to mix the seed and target deployment files in the same repo.
Maybe @browsell can comment on this. Did we recommend separated repos for seed and target clusters for IBU?

Reply:
It could be. In general, the upgradable template may not be the first version for each Z release though.
As regards to seedgen, my understanding is that the seedgen gitops setup should be a fresh copy of the target version of the real one. We can discuss whether we need to avoid upversioning the template just for upgrade defaults.